apps/namada-interface/src/slices/accounts.ts

```diff
@@ -223,7 +225,13 @@ const balancesAtom = (() => {
         set(base, { ...get(base), [address]: balance });
       });
 
-      await namada.sync();
+      const lastHeight = await namada.queryLastBlock();
+      const startHeightStorage = Number(localStorage.getItem(LATEST_SYNCED_HEIGHT_STORAGE));
+      const startHeight = startHeightStorage ? startHeightStorage : undefined;
+      await namada.sync(startHeight,lastHeight);
+      if(lastHeight){
```

Comment:
Looks like formatting is a bit off. Please run `yarn prepare` in the root of monorepo to setup husky.